etc/webapi.xml

```diff
@@ -32,7 +32,7 @@
         </resources>
     </route>
 
-    <route url="/V1/alma/payments/url" method="GET">
+    <route url="/V1/alma/payments/url/:orderId" method="GET">
```

Comment:
Is the Alma Module the only one consumers of this endpoint ?

What will happen if a merchant use this endpoint from else where and he update the module ? Is there a breaking compatibilities ? `v2` + deprecation  is needed ?

Reply:
It's magento API not Alma - it's for REST compatibility  - this road gives the payment stored in magento payment DB for an order by ID